src/marketplace/api.js

```diff
@@ -0,0 +1,58 @@
+import { axiosInstance } from '../api/entities';
+import { MARKETPLACE_URL } from './constants';
+
+const getAccessToken = () =>
+  axiosInstance.post(`${MARKETPLACE_URL}/api-security/om-auth/cloud/token`, {
+    grant_type: 'urn:ibm:params:oauth:grant-type:apikey',
+    apikey: localStorage.getItem('marketplace-key'),
+  });
+
+const query = `
+{
+  productByIds (ids: [
+    "ca54ff2ba24df884a96995f1eefd081b",
+    "ccb29628eabe15c81121ecf231a0a8cd",
+    "9e3b238934c288394680f3a53d1544e0",
+    "bd44b472123b544a630a939c650e968e"
```

Comment:
Is it possible to get these ids from API? Or do we have the static list for now? I am just thinking about other envs.

Reply:
This is just a POC, it will be definitely changed in the future. For now, we just show few selected services.